packaging/systemd/configure-ovs.sh

```diff
@@ -6,6 +6,9 @@ set -eux
 
 nmcli conn show br-ex >/dev/null && echo "br-ex already configured" && exit 0
 
+# restart NetworkManager after installing NetworkManager-ovs
+systemctl restart NetworkManager
```

Comment:
Instead of doing it here, which could potentially trigger a loop between this script that depends via systemd and NetworkManager, please let's do it in the .spec file of microshift so it's handled only once when the NetworkManager-ovs plugin is installed

```patch
diff --git a/packaging/rpm/microshift.spec b/packaging/rpm/microshift.spec
index 77069c0d..492a655e 100644
--- a/packaging/rpm/microshift.spec
+++ b/packaging/rpm/microshift.spec
@@ -213,6 +213,7 @@ fi
 # setup ovs / ovsdb optimization to avoid full pre-allocation of memory
 sed -i -n -e '/^OPTIONS=/!p' -e '$aOPTIONS="--no-mlockall"' /etc/sysconfig/openvswitch
 %systemd_post microshift-ovs-init.service
+systemctl is-active --quiet NetworkManager && systemctl restart --quiet NetworkManager || true

 %preun networking
 %systemd_preun microshift-ovs-init.service
```

Reply:
Good point, fixed.